changelog.txt

```diff
@@ -1,4 +1,8 @@
+* Fixes bug where `firebase serve --only functions` dropped the host argument.
+* Fixes bug where `grpc-js` connections used the wrong credentials.
+* Fixes bug where `firebase open` would return an error.
+* Fixes bug where profiler table rendering crashes due to undefined `path` fields.
```

Comment:
This looks like the result of a bad merge. I don't think you want the top three?